run.py

```diff
@@ -374,7 +378,30 @@ def run(args):
     log_directory = args.get("--log-dir")
 
     run_id = generate_unique_id(length=6)
-    run_dir = create_run_dir(run_id, log_directory)
+    # run_dir = create_run_dir(run_id, log_directory)
+    c = TestMetaData(
+        run_id=run_id,
+        rhbuild=rhbuild,
+        rhcs="rhcs",
```

Comment:
Why do we have `rhcs=rhcs`? Should it be the version?

Reply:
Changed rhcs to ceph Version